src/main/java/de/dataelementhub/rest/controller/v1/ExportController.java

```diff
@@ -106,17 +107,18 @@ public ResponseEntity exportStatus(
       throws Exception {
     Integer userId = DataElementHubRestApplication.getCurrentUser().getId();
     ExportInfo exportInfo = exportService.exportInfo(exportId, userId, exportDirectory);
+    String format = exportInfo.getFormat().toUpperCase();
```

Comment:
The export info method already puts the format to upper case. And when the format is set to "json" when creating the export, the file can not be found.